packages/atoms/src/index.ts

```diff
@@ -1,3 +1,5 @@
 export * from './Text';
 export * from './Box';
 export * from './Button';
+export * from './Icon/Icon';
+export * from './Icon/iconTypes';
```

Comment:
I'd prefer if this would be done in `/src/Icon/index.ts` where you could do `export * from './Icon'` and `export * from ./iconTypes` so then here you wouldn't duplicate exports from one package / component directory. End result is the same but with cleaner index file in here.

Reply:
Yes, but the argument for not using index files is pointless since it's being added in this index file anyway, right? 😃  My suggestion is just to move these exports one level deeper to improve readability of this file. This will not have a negative affect on bundle size / performance, or am I missing something?